nixos/modules/services/misc/ersatztv.nix

```diff
@@ -0,0 +1,126 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkIf
+    getExe
+    maintainers
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    ;
+  inherit (lib.types) str path bool;
+  cfg = config.services.ersatztv;
+in
+{
+  options = {
+    services.ersatztv = {
+      enable = mkEnableOption "ErsatzTV";
+
+      package = mkPackageOption pkgs "ersatztv" { };
+
+      user = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "User account under which ErsatzTV runs.";
+      };
+
+      group = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "Group under which ErsatzTV runs.";
+      };
+
+      transcodeDir = mkOption {
+        type = path;
+        default = "/var/lib/etv-transcode";
+        description = ''
+          Temporary transcode directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = path;
+        default = "/var/lib/ersatztv";
+        description = ''
+          Directory containing the server configuration files and SQLite Database file
```

Comment:
We can't split that? Config files sound like they should be under /etc/. Maybe a symlink can aid?

Reply:
The only config file is optional and not currently being created (that will change if I get [this request](https://github.com/NixOS/nixpkgs/pull/330706#discussion_r1694322938) working).

Looked into it a little more, it also puts it's caches and logs there... Splitting it out could get a little messy and deviates from upstream a little more than I'd like to keep track of. We could probably put in a request to the upstream project to have a bit more configuration of where things are. For reference, this is how the two environment variables are used: https://github.com/ErsatzTV/ErsatzTV/blob/main/ErsatzTV.Core/FileSystemLayout.cs#L66